mcpwm_foc.c

```diff
@@ -3557,6 +3558,77 @@ static THD_FUNCTION(pid_thread, arg) {
 	}
 }
 
+// This takes advantage of the IEEE 754 standard, which requires that the sign
```

Comment:
This function declaration is not in the right place in the file. Should it go higher up or lower down, though?